tools/mmp/driver.cs

```diff
@@ -499,6 +499,7 @@ static void Main2 (string [] args)
 			if (verbose > 0)
 				Console.WriteLine ("Selected target framework: {0}; API: {1}", targetFramework, IsClassic ? "Classic" : "Unified");
 
+			Log (1, $"Selected Linking: '{App.LinkMode}'");
```

Comment:
I'm technically just doing this so the test can easily tell, but it's really useful verbose info anyway, and we do it for other options as well.